tt_eager/tt_dnn/op_library/groupnorm/groupnorm_op.cpp

```diff
@@ -1839,7 +1855,7 @@ std::vector<Tensor> GroupNorm::create_output_tensors(const std::vector<Tensor> &
     } else {
         auto mem_config = this->output_mem_config;
         mem_config.shard_spec = input_tensor.shard_spec();
-        return {create_device_tensor(this->compute_output_shapes(input_tensors).at(0), program_config.out_data_format, Layout::ROW_MAJOR, input_tensor.device(), mem_config)};
+        return {create_device_tensor(this->compute_output_shapes(input_tensors).at(0), program_config.out_data_format, input_tensor.get_layout(), input_tensor.device(), mem_config)};
```

Comment:
Is input and output tensor layouts decoupled for this op? This looks like we always follow input tensor layout for output. Should the layout be from `mem_config` instead?

Reply:
I don't think mem_config has this attribute? but you are right, it should be decoupled